### PR TITLE
Inspect View Tree documentation page

### PR DESCRIPTION
**Pull request by aoleszyn**
### Description
Inspect View Tree documentation page

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -783,6 +783,7 @@ module.exports = {
             items: [
               'mobile-apps/features/device-vitals',
```
Comment (lysannep) [inline sidebars.js:784]: Although out of scope for this PR, the device-vitals.md file should be moved to the mobile-app-diagnostics folder like the other files in same structure.

File: docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+---
+id: view-tree
+title: Inspect View Tree
+sidebar_label: Inspect View Tree
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Inspect View Tree is a functionality that is designed to help you discover all the visible elements of your mobile application while developing your test scripts. It captures a snapshot of the user interface and allows you to navigate and inspect view objects of your native apps.
```
Comment (lysannep) [inline docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md:8]:
```suggestion
Inspect View Tree is a feature that is designed to help you discover all the visible elements of your mobile application while developing your test scripts. It captures a snapshot of the user interface and allows you to navigate and inspect view objects of your native apps.
```

File: docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+Inspect View Tree feature allows you to see, debug, analyze or determine the state of individual View components.  It also informs you about global changes in the view tree.
```
Comment (lysannep) [inline docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md:10]:
```suggestion
Inspect View Tree feature allows you to see, debug, analyze or determine the state of individual View components. It also informs you about global changes in the view tree. 
```

File: docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+The following table lists the available attributes:
+
+|Name|Description|
+|:---|:---|
+|ID|The class identifier|
+|Visibility| A value that checks if the component is present in the View Tree|
+|Focusable|A value that provides additional information associated with accessibility|
+|Alpha|A value that describes how translucent or opaque the component is|
+|Enabled|Checks the interactivity of the component|
+|ContentDesc|The accessibility description of an UI component|
```
Comment (lysannep) [inline docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md:79]:
```suggestion
|ContentDesc|The accessibility description of a UI component|
```

File: docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+|Text|A text attribute of an UI component|
```
Comment (lysannep) [inline docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md:80]:
```suggestion
|Text|A text attribute of a UI component|
```

File: docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+## Limitations
+
+:::note
+* Webviews and hybrid apps will have additional capture limitations.
```
Comment (lysannep) [inline docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md:85]: What are these limitations?

**Review by lysannep — CHANGES_REQUESTED** [review]
Looks good to me! ✨ Added a few minor suggestions.

File: docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+The Inspect View Tree feature works for all the automated test framework types.
+:::
+
+## Accessing Inspect View Tree
+
+You can access the Inspect View Tree from the [Live Test Results](/mobile-apps/live-testing/live-mobile-app-testing/) or [Automated Test Results](/mobile-apps/automated-testing/) page.
+
+1. In Sauce Labs, click **LIVE** or **AUTOMATED** and then click **Test Results**.  
+<br/><img src={useBaseUrl('img/mobile-apps/DEvice-Vitals-screenshot-2.png')} alt="Mobile app settings navigation" width="200"/>
+<br/><img src={useBaseUrl('img/mobile-apps/DEvice-Vitals-screenshot-7.png')} alt="Mobile app settings navigation" width="200"/>
```
Comment (lysannep) [inline docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md:48]:
```suggestion
<br/><img src={useBaseUrl('img/mobile-apps/Device-Vitals-screenshot-7.png')} alt="Mobile app settings navigation" width="200"/>
```

File: docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md (hunk 6 above)
Comment (aoleszyn) [reply docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md:85]: @lysannep I spoke with @kristofmuhi   about mentioned limitations and the team does not know which exact details are missing from the capture of webviews, they know that some buttons and views might be missing, but this information is hidden. So the selected sentence under "limitations" does not include a detailed explanation.

**Review by lysannep — APPROVED** [review]

File: docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+2. Select a test result from the list.
+<br/><img src={useBaseUrl('img/mobile-apps/view-tree-screenshot-1.png')} alt="Mobile app settings navigation" width="800"/>
+
+3. Click on **Inspect** button.
+<br/><img src={useBaseUrl('img/mobile-apps/view-tree-screenshot-2.png')} alt="Mobile app settings navigation" width="600"/>
+
+The Inspect View Tree feature will display the captured snapshots of detectable view changes:
+
+<br/><img src={useBaseUrl('img/mobile-apps/view-tree-screenshot-3.png')} alt="Mobile app settings navigation" width="700"/>
```
Comment (kristofmuhi) [inline docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md:58]: Can we change the screenshot to this one? https://saucelabs.slack.com/files/U0166MC9HRU/F03LE6PFU3Z/image.png

File: docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
+---
+id: view-tree
+title: Inspect View Tree
+sidebar_label: Inspect View Tree
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Inspect View Tree is a feature that is designed to help you discover all the visible elements of your mobile application while developing your test scripts. It captures a snapshot of the user interface and allows you to navigate and inspect view objects of your native apps.
```
Comment (wswebcreation) [inline docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md:8]: @kristofmuhi 
Small but maybe important note here. Can we add a note here that the view tree/ui hierarchy shown here is NOT the same as with Appium. They can't copy over the XPATH to find elements. Appium shows XML, this is not showing XML. Can we also add that the way Appium collects and translates the data is different in comparison to ours, but emphasise that our solution will help them debug if that element was actually in the UI Hierarchy

File: docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md (hunk 8 above)
Comment (aoleszyn) [reply docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md]: Done :)

File: docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md (hunk 9 above)
Comment (kristofmuhi) [reply docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md:8]: Agreed! @aoleszyn  lets publish the doc in this form, overall looks good, and create a new PR for Wim's comments and figure out how to add this to the disclaimer part below

**Pull request merged**